### PR TITLE
Fix for #25 - Set json ascii encoding to false

### PR DESCRIPTION
**Pull request by avinashvarna**

### CODE REVIEW

File: dhatu_restplus.py [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 
 app = Flask(__name__)
+app.config['JSON_AS_ASCII'] = False
```
Comment (vvasuki) [inline dhatu_restplus.py]: एवमपि शक्यत इत्य् आश्चर्यम्!

**Comment by avinashvarna** [comment]
यदि jsonify इत्यत्र परिवर्तनीयं, तर्हि अयमेव मार्गः । यतो हि jsonify इत्यस्य ensure_ascii इति keyword argument नास्ति । https://stackoverflow.com/a/39561607 इत्यस्माज्ज्ञातम् ।

**Comment by vvasuki** [comment]
> यदि jsonify इत्यत्र परिवर्तनीयं, तर्हि अयमेव मार्गः । यतो हि jsonify इत्यस्य ensure_ascii इति keyword argument नास्ति । https://stackoverflow.com/a/39561607 इत्यस्माज्ज्ञातम् ।

अहमपि तदैवापश्यम्। यदि साधु स्मरामि jsonify internally passes all keyword arguments to dumps. किन्त्व् एतदेव वरीयः।

@drdhaval2785 स्वीक्रियताम् परिवर्तनम् यदि नास्ति विप्रतिपत्तिः।

**Comment by drdhaval2785** [comment]
I guess I broke something on server. Will fix tomorrow. API may be off today.

**Comment by avinashvarna** [comment]
> jsonify internally passes all keyword arguments to dumps.

Yes, but the behavior is different, it converts it to a dict before passing (https://stackoverflow.com/a/16003910). So my understanding is that the kwargs will not impact the behavior of dumps, but will go into the json output. i.e. jsonify(x, ensure_ascii=False) may produce {"ensure_ascii": false, <contents of x>}

**Comment by drdhaval2785** [comment]
@avinashvarna and @vvasuki

The API is functional now. But the UI of Avinash seems to hang indefinitely. Can you check up whats the issue there?

**Comment by drdhaval2785** [comment]
And the PR seems to have done the job. Now response is quite human readable in API also.

**Comment by avinashvarna** [comment]
I merged your PR and the UI seems to work fine now. Is v0.0.3 the right version to use in the server? E.g. 
https://api.sanskritworld.in/v0.0.2/verbforms/slp1/corayAmAsa?output_transliteration=devanagari gives a 500 error, but
https://api.sanskritworld.in/v0.0.3/verbforms/slp1/corayAmAsa?output_transliteration=devanagari returns some valid data

Can we have a /latest/ that always points to the correct version, so that the UI does not have to keep changing if the only thing that changed is the version?

**Comment by drdhaval2785** [comment]
v0.0.3 has not started yet. What you see as data is default error message.
It shows valid URIs to use.

**Comment by avinashvarna** [comment]
In that case, please check why a form not in the database is causing a 500 error. I suspect that the python-prakriya package does not handle the case of a missing form correctly.

**Pull request merged**